src/components/InQueueMessaging/InQueueCallbackComponent.js

```diff
@@ -74,9 +76,8 @@ class InQueueCallbackComponent extends React.Component {
     //  get instance of Flex manager
     let mgr = Flex.Manager.getInstance();
 
-    const baseUrl = process.env.REACT_APP_SERVICE_BASE_URL;
-    console.log("==========" + baseUrl);
-    let temp = transferTask(baseUrl + '/inqueue-utils', {
+    console.log(buildUrl('/inqueue-utils'));
+    let temp = transferTask(buildUrl('/inqueue-utils'), {
```

Comment:
No `let temp` is required. Can just have `transferTask(buildUrl('/inqueue-utils'), ...).then...`

There are a few other instances of this as well.